### ISSUE TITLE
can you please clarify , what you mean under feature distributions 

### ISSUE THREAD

**Issue by Sandy4321** [sljordan91/naive-bayes#1]
can you please clarify , what you mean under feature distributions 

for example
discrete features that are categorically distributed. The categories of each feature are drawn from a categorical distribution.
https://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.CategoricalNB.html#sklearn.naive_bayes.CategoricalNB

seems to be you mean that distributions are estimated from data?
for given categorical feature it is probabilities and conditional probability (values and target) from calculated from data?

as mentioned in
https://datascience.stackexchange.com/questions/58720/naive-bayes-for-categorical-features-non-binary
Some people recommend using MultinomialNB which according to me doesn't make sense because it considers feature values to be frequency counts

**Comment by sljordan91** [comment]
That's right - the distributions of categorical features are computed directly from the training set [(see here)](https://github.com/sljordan91/naive-bayes/blob/master/naive-bayes/naive_bayes.py#L110). There's no assumption that it follows a binomial or multinomial distribution at all.

**Comment by Sandy4321** [comment]
conditional probability (values and target) from calculated from data?

Do you calculate conditional probability (values and target)  from data As well?
In any case do you treat properly dependency between features?
So it will be not so Naive Bayesian?
